src/WorkflowsRules/Actions/ADF.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Kanvas\Packages\WorkflowsRules\Actions;
+
+use GuzzleHttp\Client;
+use Kanvas\Packages\WorkflowsRules\Contracts\Interfaces\ActionInterfaces;
+use Kanvas\Packages\WorkflowsRules\Contracts\Interfaces\WorkflowsEntityInterfaces;
+use Phalcon\Di;
+use Throwable;
+
+class ADF implements ActionInterfaces
+{
+    protected ?string $message = null;
+
+    protected ?array $data = [];
+
+    protected int $status = 1;
+
+    /**
+     * handle.
+     *
+     * @param  WorkflowsEntityInterfaces $entity
+     * @param  array $params
+     *
+     * @return array
+     */
+    public function handle(WorkflowsEntityInterfaces $entity, array $params = []) : array
+    {
+        $response = null;
+        $di = Di::getDefault();
+        try {
+            $data = $entity->getAll();
+            $request = [
+                'name' => $data['firstname'],
+                'lastname' => $data['lastname'],
+                'phone' => $data['phone'],
+                'message' => $data['message'],
+                'email' => $data['email'],
+                'username' => key_exists('username', $data) ? $data['username'] : null,
```

Comment:
key_exists('username', $data) ? $data['username'] : null, isnt this the sames as $data['username'] ?: null ?